static/app/components/gridEditable/index.tsx

```diff
@@ -415,7 +415,7 @@ class GridEditable<
     return (
       <GridRow>
         <GridBodyCellStatus>
-          <EmptyStateWarning>
+          <EmptyStateWarning small>
```

Comment:
default size was too large to fit in the same box size as our widget

Reply:
Does this impact other places around the app that use gridEditable?